sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com.azure.security.keyvault.administration/SelectiveRestore.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.administration;
+
+import com.azure.core.util.polling.SyncPoller;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.security.keyvault.administration.models.KeyVaultRestoreOperation;
+
+/**
+ * This sample demonstrates how to selectively restore a key from key vault backup.
+ */
+public class SelectiveRestore {
+    /**
+     * Authenticates with the key vault and shows how to selectively restore a key from key vault backup.
+     *
+     * @param args Unused. Arguments to the program.
+     * @throws IllegalArgumentException when an invalid key vault URL is passed.
+     */
+    public static void main(String[] args) {
+        // Instantiate an KeyVaultBackupClient that will be used to call the service. Notice that the client is using
+        // default Azure credentials. To make default credentials work, ensure that environment variables
+        // 'AZURE_CLIENT_ID', 'AZURE_CLIENT_KEY' and 'AZURE_TENANT_ID' are set with the service principal credentials.
+
+        // To get started, you'll need a URI to an Azure Key Vault. See the README (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Azure.Security.KeyVault.Administration/README.md)
+        // for links and instructions.
+        KeyVaultBackupClient backupClient = new KeyVaultBackupClientBuilder()
+            .vaultUrl("https://{YOUR_VAULT_NAME}.vault.azure.net")
+            .credential(new DefaultAzureCredentialBuilder().build())
+            .buildClient();
+
+        // Using the KeyVaultBackupClient, you can restore a single key from backup by key name. The data source for a
+        // selective key restore is a storage blob accessed using Shared Access Signature authentication.
+        String keyName = "<key-name>";
+        String blobStorageUrl = "<blob-storage-url>";
+        String sasToken = "<sas-token>";
+        String folderName = "<folder-name>";
+
+        SyncPoller<KeyVaultRestoreOperation, Void> restorePoller =
+            backupClient.beginSelectiveRestore(keyName, blobStorageUrl, sasToken, folderName);
+    }
```

Comment:
Shouldn't you wait for it, for consistency with other examples if nothing else?

Reply:
You are right, I must have missed it :)